test/performance/odp_packet_gen.c

```diff
@@ -1032,6 +1068,27 @@ static int close_pktios(test_global_t *global)
 	return ret;
 }
 
+static inline void get_timestamp(odp_packet_t pkt, uint32_t ts_off, rx_lat_data_t *lat_data)
+{
+	ts_data_t ts_data;
+	uint64_t nsec;
+
+	if (odp_unlikely(odp_packet_copy_to_mem(pkt, ts_off, sizeof(ts_data), &ts_data) < 0 ||
+			 odp_be_to_cpu_64(ts_data.magic) != TS_MAGIC))
+		return;
+
+	nsec = odp_time_global_ns() - odp_be_to_cpu_64(ts_data.tx_ts);
```

Comment:
The endianness conversions could be dropped for application internal data to minimize overhead.

Reply:
Sure, I guess there are no valid use-cases where the swap would be needed. V2.